docs/specs/meta-clients/web3wallet/sdk-api.md

```diff
@@ -71,7 +71,7 @@ class Web3Wallet {
   public abstract getPendingAuthRequests(): Promise<Record<number, PendingRequest>>;
   
   // register device token for Push server (BOTH)
-  public abstract registerDeviceToken(token: string): Promise<void>;
+  public abstract registerDeviceToken(token: string, isDecrypted: boolean?): Promise<void>;
```

Comment:
I think we should flip this language actually for clarity. Notifications were always un-encrypted, now they are becoming encrypted. We are registering the device token to receive encrypted notifications, not decrypted ones.

```suggestion
  public abstract registerDeviceToken(token: string, enableEncrypted: boolean?): Promise<void>;
```

Reply:
Yeah from the server perspective, it makes more sense. I'm fine with changing